debian/control

```diff
@@ -14,7 +14,6 @@ Depends: ${shlibs:Depends},
  parted,
  e2fsprogs,
  libraspberrypi-bin,
- net-tools,
```

Comment:
You should add iproute2 as dependency. Yes iproute2 is in the default installation of debian, but it should be in the dependency of this package.

Reply:
good catch, I will add it.
but it's still not clear to me how the /usr/bin/ip get installed.
```
pi@RevPi2222:~ $ dpkg -L iproute2 | grep -w ip | grep -v ip-
/bin/ip
/usr/share/man/man8/ip.8.gz
/sbin/ip
pi@RevPi2222:~ $ which ip
/usr/sbin/ip
pi@RevPi2222:~ $ whereis ip
ip: /usr/bin/ip /usr/sbin/ip /usr/share/man/man7/ip.7.gz /usr/share/man/man8/ip.8.gz
pi@RevPi2222:~ $ ls -l /usr/bin/ip
-rwxr-xr-x 1 root root 503112 Dec  3  2020 /usr/bin/ip
pi@RevPi2222:~ $ ls -l /usr/sbin/ip
lrwxrwxrwx 1 root root 7 Dec  3  2020 /usr/sbin/ip -> /bin/ip
pi@RevPi2222:~ $ ls -l /sbin/ip
lrwxrwxrwx 1 root root 7 Dec  3  2020 /sbin/ip -> /bin/ip
pi@RevPi2222:~ $ ls -l /bin/ip
-rwxr-xr-x 1 root root 503112 Dec  3  2020 /bin/ip
```

Reply:
The iproute2 package installs it to `/bin/ip`. /bin is a symlink to `/usr/bin`.

Reply:
good to know, thanks